airflow/api_connexion/openapi/v1.yaml

```diff
@@ -585,6 +585,41 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+  /dags/{dag_id}/setTaskInstanceNote:
```

Comment:
It should be `PATCH /dags/{dag_id}/`. This will allow several changes to be made with a single HTTP request.

Reply:
Indeed, should have made this a PATCH operation.
However, this ties in closer with a DAGRun than a DAG in general IMO.
Shall I make it part of the already existing patch operation in `/dags/{dag_id}/dagRuns/{dag_run_id}:`?
https://github.com/apache/airflow/blob/527c90c9804ee66cfa800381cbd67280e5d3a8a9/airflow/api_connexion/openapi/v1.yaml#L787

Reply:
Or even better probably, make a patch operation on this endpoint?
 `/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}:`
https://github.com/apache/airflow/blob/527c90c9804ee66cfa800381cbd67280e5d3a8a9/airflow/api_connexion/openapi/v1.yaml#L1193

Reply:
We'd want both `/dags/{dag_id}/dagRuns/{dag_run_id}` and `/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}` right?

I would've suggested extending `/dags/{dag_id}/updateTaskInstancesState` but the url name doesn't really work.

Reply:
Yes to me that makes sense. I'll change it and see if I can get somewhere with the front-end soon. Thanks for the early feedback